From 2a381dc128550dbf836f72ffaec1ce78c9436474 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Fri, 28 Aug 2015 09:37:13 +0200
Subject: [PATCH] improve default config

remove bootchart and floppy from default config and add mmc
---
 mkinitfs.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mkinitfs.conf b/mkinitfs.conf
index 35cc96a..67a5444 100644
--- a/mkinitfs.conf
+++ b/mkinitfs.conf
@@ -1,2 +1,2 @@
 # run mkinitfs -L for a list of available features
-features="ata base bootchart cdrom cramfs ext2 ext3 ext4 floppy keymap kms raid scsi usb virtio"
+features="ata base cdrom ext2 ext3 ext4 keymap kms mmc raid scsi usb virtio"
-- 
2.5.0

